ui/templates/welcome.html

```diff
@@ -3,7 +3,11 @@
     <div class="section-header">
         <div class="row">
             <div class="col-md-2 col-md-offset-5">{% if perms.learningresources.add_repository %}<a href="{% url 'create_repo' %}" role="button"
-                                                     class="btn btn-primary btn-large">Create repository</a>{% endif %}</div>
+                                                     class="btn btn-primary btn-large">Create repository</a>
```

Comment:
if you want to keep the nice indentation, you an use the `{% spaceless %}` tag.
[look here](https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#spaceless)